build/MSBuildToolset/build-toolset.ps1

```diff
@@ -0,0 +1,71 @@
+﻿# Builds the toolset for use in unix msbuild-coreclr
```

Comment:
Can we change the powershell script to have the following pattern?

``` ps1
$ErrorActionPreference="Stop"

try
{
   # Code that you have today 
catch
{
    write-host "Error: $($_.Exception.Message)"
    exit 1
}
```

The issue with powershell is that it essentially does no error checking.  If `dotnet` failed to exist or restore simply failed the script would print an error message and continue on.  Eventually exiting with almost certainly a 0 exit code indicating success.  The above pattern helps lock down powershell scripts and prevent this from happening.